react/features/conference/middleware.ts

```diff
@@ -46,6 +71,20 @@ MiddlewareRegistry.register(store => next => action => {
 
         break;
     }
+
+    case CONFERENCE_LEFT: {
+        clearInterval(intervalId);
+        intervalId = null;
+
+        break;
+    }
+
+    case CONFERENCE_FAILED: {
```

Comment:
Nit: use a fall through, so the code is not repeated. That is, the 2 case statements together, since if there is no break it will continue to the next.

Reply:
Thanks for the hint!